packages/apps/human-app/server/src/common/utils/pageable-classes.ts

```diff
@@ -1,10 +1,10 @@
 import { ApiPropertyOptional } from '@nestjs/swagger';
 import { Type } from 'class-transformer';
 import { IsEnum, IsNumber, IsOptional, Max, Min } from 'class-validator';
-import { SortOrder } from '../enums/global-common.interface';
+import { SortOrder } from '../enums/global-common';
 import { AutoMap } from '@automapper/classes';
 
-export abstract class PageableDto {
```

Comment:
I don't like this naming and actually don't agree with the reviewer: for me some of data structures are peagable and as such adhere to pageable interface, if change must be made then I would just change this from class to interface but as far as I know there will be problem with decorathors then, @BartoszSolkaBD what is your opinion about this?

Reply:
fix: dd675f80